### PR TITLE
Add nobl9 indicator kind annotation support. 

### PR DESCRIPTION
**Pull request by fpiwowarczyk**
# Summary
This PR tires to resolve problem from #194.  

This problem begin because OpenSLO spec doesn't have substitute of Indicator kind: Direct/Agent. 

To resolve this issue I added new annotation for indicators. From now on if you wan't to convert your indicator to be Nobl9 Metric Source Direct you need to specify annotation in metadata as in below example 

``` yaml 
 indicator:
    metadata:
      name: someName
      annotations:
        nobl9.com/indicator-kind: Direct
    spec:
    .
    .
    .
```

Will give you ourput 
``` yaml
    indicator:
        metricSource:
            project: default
            name: someName
            kind: Direct
     .
     .
     .
```

Same thing for kind Agent but it is set as default kind.

### CODE REVIEW

File: test/v1/slo/slo-with-annotations.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+apiVersion: openslo/v1
+kind: SLO
+metadata:
+  name: monthy-openslo-slo
+  displayName: Python
+spec:
+  #annotation:nobl9-kind:Direct
```
Comment (n0npax) [inline test/v1/slo/slo-with-annotations.yaml:7]: I'm afraid this won't cover my use-case.
Annotation is currently stored as a comment in yaml which can't be generated using SLO structs.

Would it be possible to replace `#annotation:nobl9-kind:Direct` with
```yaml
metadata:
  annotations:
    nobl9-kind: Direct
```

---

Just a bit more about use-case I've.
I've plenty of services which are quite similar, so for example availability `SLOs` are almost the same for each of the apps I've. I'm not crafting any of openSLO files manually, but I'm generating those using go structs.  This makes it important for me to be able to generate relevant config using structs which can be mapped directly to `oslo` yaml files.

Comment (fpiwowarczyk) [reply test/v1/slo/slo-with-annotations.yaml:7]: Yes, it looks like better solution :D my bad! I forget about that option, I'll prepare something else in couple of days

**Comment by fpiwowarczyk** [comment]
@n0npax You can take look again ;) I think it is better now

File: internal/pkg/convert/convert.go [hunk 2]
```diff
@@ -303,11 +314,23 @@ func getN9Indicator(i v1.SLISpec, project string) nobl9v1alpha.Indicator {
...
+func getKindFromAnnotations(metadata v1.Metadata) string {
+	if value, ok := metadata.Annotations[n9KindAnnotation]; ok {
+		switch strings.ToLower(value) {
+		case "direct":
+			return kindDirect
+		case "agent":
+			return kindAgent
+		}
+	}
+	return kindAgent
```
Comment (n0npax) [inline internal/pkg/convert/convert.go:331]: do you believe user should be warned here? or do you believe relevant docs would be enough?

**Review by n0npax — DISMISSED** [review]
Looks good, Thanks!

I believe it would be nice to have relevant docs about this change, but current state is more than enough for me to proceed. Thanks again!

**Pull request merged**